2025-08-03T14:24:05.554700+00:00 CRITICAL 'ansible_navigator.actions.run_playbook._handle_message' Unhandled message from runner queue, discarded: {'event': 'verbose', 'uuid': '1c9dee27-e023-4cfb-84f0-424c073d8538', 'counter': 1, 'stdout': '\x1b[0;31mERROR! the playbook: /home/ansible/vars/playbook2.yml could not be found\x1b[0m', 'start_line': 0, 'end_line': 1, 'runner_ident': 'f3e6c4d5-b7f2-471f-9515-a69f00d704a2', 'created': '2025-08-03T14:24:05.541762'}
